templates/legal/terms-and-policies/launchpad-terms-of-service.html

```diff
@@ -0,0 +1,103 @@
+{% extends "legal/_base_legal.html" %}
+
+{% block title %}Launchpad terms of service{% endblock %}
+
+{% block content %}
+
+<div class="p-strip is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h1 id="title">Launchpad terms of service</h1>
+
+      <section>
+        <h2 id="trademark-and-logo">Trademark and logo licence</h2>
+        <p>"<em>Launchpad</em>" and the Launchpad "Gem" logo are trademarks of Canonical Ltd.</p>
+        <p>The Launchpad "Gem" logo was created by <a href="https://launchpad.net/~etretyak">Eugene Tretyak</a> as part of the <a href="https://help.launchpad.net/logo">Launchpad Logo Contest</a>. The copyright in the Launchpad logo is owned by Canonical Ltd. and is licensed under a <a href="http://creativecommons.org/licenses/by-nd/2.0/uk/">Creative Commons Attribution-No Derivative Works 2.0 UK: England & Wales License</a>.</p>
```

Comment:
@cjwatson all external links need to be decorated with a `class="p-link--external"`